src/device/usbd.c

```diff
@@ -817,10 +817,11 @@ static bool process_get_descriptor(uint8_t rhport, tusb_control_request_t const
         // The 0xEE index string is a Microsoft OS Descriptors.
         // https://docs.microsoft.com/en-us/windows-hardware/drivers/usbcon/microsoft-defined-usb-descriptors
         return false;
-      }else
+      }
+      else
       {
         uint8_t const* desc_str = (uint8_t const*) tud_descriptor_string_cb(desc_index, p_request->wIndex);
-        TU_ASSERT(desc_str);
+        TU_VERIFY(desc_str);
```

Comment:
like above, host is well behaved, if it request out of bound descriptors, it is usually device  report it has that one by mistake. This help troubleshooting using assert to print out the error in the LOG